plugin/openassetio_manager_bal/bal.py

```diff
@@ -85,6 +88,15 @@ def validate_settings(settings: dict):
 
     defaults = make_default_settings()
 
+    if SETTINGS_KEY_LIBRARY_PATH in settings:
+        if not isinstance(settings[SETTINGS_KEY_LIBRARY_PATH], str):
+            raise ValueError(f"{SETTINGS_KEY_LIBRARY_PATH} must be a str")
+    if SETTINGS_KEY_SIMULATED_QUERY_LATENCY in settings:
+        if not isinstance(settings[SETTINGS_KEY_SIMULATED_QUERY_LATENCY], int):
+            raise ValueError(f"{SETTINGS_KEY_SIMULATED_QUERY_LATENCY} must be an int")
+        if settings[SETTINGS_KEY_SIMULATED_QUERY_LATENCY] < 0:
+            raise ValueError(f"{SETTINGS_KEY_SIMULATED_QUERY_LATENCY} must be greater than zero")
```

Comment:
[fix] It can be 0 to disable the simulated latency.

Reply:
Change to "must not be negative"
8406912a813998061a29653503479c59ca705732